### DISCUSSION TITLE
İbax sizce kaç dolar olucak

### DISCUSSION THREAD

**Discussion by Emircan360** [IBAX-io/go-ibax#1310] [Soru Cevap - Türkçe]
İbax sizce kaç dolar olucak

**Comment by ahmederoni** [comment]
Kısa vadede fiyatına bakmazsızın köşeye atıp unutmak lazım herkes sol ve avax ile kıyaslıyor o kadar olacagına ihtimal dahi vermiyorum,yinede ne olacagı belli olmaz diye köşede dursun

**Reply by Kriptoportakal** [reply]
bunu bilemeyiz ibaxta avax gibi olmasada bi 20 dolar hakkıdır boğada

**Reply by Azizatli21** [reply]
Bence 100 dolar olur

**Comment by Yusufknk** [comment]
Bunu ilerleyen zamanlarda birlikte tecrübe edeceğiz.

**Comment by nescem** [comment]
1 Şubat'ta ne olacak çok merak ediyorum

**Comment by kublaycelik** [comment]
Çıkış fiyatıda önemli. Talep önemli. Ama cent bazında kalmaz diye düşünüyorum. 10-30 dolar arası bir aralıkta olur. Uzun vadede ne olur tabi orası bilinmez. 4-5 Borsa listelenmesi, bir kaçta sağlam şirketlerle ortaklik falan duyurursa ozaman uçar gider.

**Comment by natolu2738** [comment]
Hedef süreyi belirtmiş olsaydınız. Ona göre bir tahminde bulunurdum. Ama yine de orta vade hedefim (İlk 1 yıl içinde) 10 $
